README.md

```diff
@@ -151,3 +151,39 @@ To test the NGINX Kubernetes Gateway run:
 ```
 make unit-test
 ```
+
+# Release Process for NGINX Kubernetes Gateway
+
+### NGINX Kubernetes Gateway will use semantic versioning for releases described in semver.org.. 
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
```

Comment:
I don't think we need to copy and paste their wording in our doc and this doesn't need to be a whole section.

Our use of semver is just a statement somewhere.

"NGINX Kubernetes Gateway uses semantic versioning for its releases. For more information see https://semver.org.

Warning: Major version zero (0.y.z) is reserved for development, anything MAY change at any time. The public API is not stable."

Reply:
Thanks!  I made this update.